src/Volume.ts

```diff
@@ -140,22 +139,20 @@ export default class Volume {
   private pixel_size: [number, number, number];
   /* eslint-enable @typescript-eslint/naming-convention */
 
-  constructor(imageInfo?: ImageInfo) {
+  constructor(imageInfo: ImageInfo = getDefaultImageInfo()) {
```

Comment:
are you sure that `getDefaultImageInfo` gets called every time you do a `new Volume()` ?

Reply:
Okay, looks like we're safe: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters#evaluated_at_call_time

Reply:
great! thank you for checking that.. I was definitely thinking of the Python thing I had read recently.